Resources/views/MediaAdmin/edit.html.twig

```diff
@@ -50,8 +67,7 @@ file that was distributed with this source code.
                             <table class="table">
                                 <tr>
                                     <th>{{ 'label.size'|trans({}, 'SonataMediaBundle') }}</th>
-                                    <td>{{ object.width|number_format_decimal }}px x {{ object.height|number_format_decimal }}px
-                                        {% if object.size > 0 %}({{ object.size|number_format_decimal }}o){% endif %}</td>
+                                    <td>{{ object.width }} x {{ object.height }} {% if object.size > 0 %}({{ macros.bytesToSize(object.size) }}){% endif %}</td>
```

Comment:
Not sure about what you said of `number_format`. It uses php: http://php.net/manual/es/function.number-format.php. And there is nothing about automatic configuration based on the locale.